modules/nav.js

```diff
@@ -0,0 +1,56 @@
+const list = document.getElementById('list');
+const addNew = document.getElementById('addNew');
+const contact = document.getElementById('contact');
+export const logo = document.getElementById('logo');
+const addSection = document.getElementById('addSection');
+const listSection = document.getElementById('listSection');
+const contactSection = document.getElementById('contactSection');
+const home = document.getElementById('homePage');
+
+export class Navigation {
+  constructor() {
+    this.addSection = document.getElementById('addSection');
+    this.listSection = document.getElementById('listSection');
+    this.contactSection = document.getElementById('contactSection');
+    this.home = document.getElementById('homePage');
+  }
+
+  homePage() {
+    this.addSection.style.display = 'none';
+    this.listSection.style.display = 'none';
+    this.contactSection.style.display = 'none';
+    this.home.style.display = 'block';
+    this.list.style.color = 'black';
+    this.addNew.style.color = 'black';
+  }
+}
+
```

Comment:
- [ ] This project is a restructure of the previous one and so should behave similarly. When I add some books and reload the page, the books disappear (see images below). I have to add another book or click **List** on the navbar for the books in the localStorage to be loaded. I would recommend you ensure your books are listed upon page refresh.

  **SUGGESTION** You can add an event listener for page load and then add the block CSS property to the listSection element as below:
  ```
  window.addEventListener('load', () => {
    listSection.style.display = 'block'
  })
  ```


  Book is added | Page reloaded | Another book is added
  :---:|:---:|:---:
  ![moise-1](https://user-images.githubusercontent.com/36542986/155968593-a05355f8-aebe-4485-b954-9c90da91e4f8.jpg) | ![moise-2](https://user-images.githubusercontent.com/36542986/155968602-61e1ecfb-5657-4ea8-b611-7b7bfd017ee6.jpg) | ![moise-3](https://user-images.githubusercontent.com/36542986/155968607-58dd9ed7-96a1-4c3a-b359-db6029103cd1.jpg)